drv/update-api/src/lib.rs

```diff
@@ -173,6 +178,31 @@ pub enum SlotId {
     B,
 }
 
+impl From<RotSlot> for SlotId {
```

Comment:
Note to the reader: I did this to transition away from using `RotSlot` which comes from stage0_handoff to using the higher level `SlotId`